app/components/views/GetStartedPage/DaemonLoading/Form.js

```diff
@@ -101,6 +102,19 @@ const DaemonLoadingBody = ({
             </div>
           }
         </div>
+        {daemonWarning && getCurrentBlockCount <= 0 ?
+          <Aux>
+            <div className="get-started-last-log-lines">
+              <div className="last-dcrwallet-log-line">{daemonWarning}</div>
+            </div>
+            <div className="advanced-page-form">
+              <div className="advanced-daemon-row">
+                <T id="getStarted.longWaitWarning" m="You are currently upgrading to dcrd version 1.4.0.  Typically, this one-time reindexing take 30-45mins on an average machine."/>
```

Comment:
I'd remove the reference to a specific version, given that this might happen in newer versions as well:
```suggestion
                <T id="getStarted.longWaitWarning" m="You are currently upgrading to a new dcrd version.  Typically, this one-time reindexing takes 30-45 minutes on an average machine."/>
```